iothub/device/src/PlugAndPlayConvention/PnpConventionHelper.cs

```diff
@@ -80,6 +93,30 @@ public static string CreatePropertyPatch(string propertyName, string serializedP
         /// Create a key-embedded value property patch for read-write properties.
         /// Embedded value property updates are sent from a device in response to a service-initiated read-write property update.
         /// </summary>
+        /// <remarks>
+        /// For a root-level property update, the patch is in the format:
+        ///     {
+        ///         "samplePropertyName": {
+        ///             "value": 20,
+        ///             "ac": 200,
+        ///             "av": 5,
+        ///             "ad": "The update was successful."
+        ///         }
+        ///     }
+        ///
+        /// For a component-level property update, the patch is in the format:
+        ///     {
+        ///         "sampleComponentName": {
+        ///             "__t": "c",
+        ///             "samplePropertyName": {
+        ///                 "value": 20,
+        ///                 "ac": 200,
+        ///                 "av": 5,
+        ///                 "ad": "The update was successful."
+        ///             }
+        ///         }
+        ///     }
+        /// </remarks>
         /// <param name="propertyName">The property name, as defined in the DTDL interface.</param>
         /// <param name="serializedPropertyValue">The serialized property value, in the format defined in the DTDL interface.</param>
         /// <param name="ackCode">The acknowledgment code from the device, for the embedded value property update.</param>
```

Comment:
For these property patches, it is going to be easy to break.

Minimally they'll need unit tests that get the output and deserialize.

However, we should probably consider just making classes to represent these objects, initialize them, and serialize.

Reply:
Added a task for this.